controllers/activemqartemis_reconciler.go

```diff
@@ -2506,7 +2509,7 @@ func getDeploymentCondition(cr *brokerv1beta1.ActiveMQArtemis, podStatus olm.Dep
 	if reconcileError != nil {
 		return metav1.Condition{
 			Type:    brokerv1beta1.DeployedConditionType,
-			Status:  metav1.ConditionUnknown,
+			Status:  metav1.ConditionFalse,
```

Comment:
This is an interesting change, it will force our Ready condition to false. I think that is correct, Ready is about the CR desired state being respected. Any failure to apply/update the api server, even for transient failures should make it false. Otherwise, with the shortcut Ready condition being true, there is no good reason to peek at the rest of the status conditions and this one my well be important.

Reply:
the changes seems to be an auto generated one, will rollback

Reply:
seems it's because of the "git --amend" argument, I have remove the argument and rollback.